docs/getting-started/install-scylla/oss-install-gcp.rst

```diff
@@ -0,0 +1,92 @@
+.. |VERSION| replace:: 5.2
+.. |IMAGE-NAME| replace:: scylladb-5-2-1
+.. |IMAGE-ID| replace:: 8305314263548747356
+.. |PROJECT-NAME| replace:: scylla-images
+
+==========================
+Launch ScyllaDB on GCP
+==========================
+
+This article will guide you through self-managed ScyllaDB deployment on GCP. For a fully-managed deployment of ScyllaDB 
+as-a-service, see `ScyllaDB Cloud documentation <https://cloud.docs.scylladb.com/>`_.
+
+With the right setting, ScyllaDB Image can be used for production. For the ScyllaDB Image documentation, see 
+the `ScyllaDB Machine Image project <https://github.com/scylladb/scylla-machine-image>`_.
+
+Prerequisites
+----------------
+
+* Active GCP account
+* `Google SDK <https://cloud.google.com/sdk/docs/install>`_, which includes the ``gcloud`` command-line tool
+* ScyllaDB Image requires at least 2 vCPU servers.
+
+ScyllaDB |VERSION| Image Information
+---------------------------------------
+
+* Image name: |IMAGE-NAME|
+* Image ID: |IMAGE-ID|
+* Project name: |PROJECT-NAME|
+
+
+Launching ScyllaDB on GCP
+------------------------------
+
+#. Choose an instance type. See :ref:`Cloud Instance Recommendations for GCP <system-requirements-gcp>` for the list of recommended instances.
+    
+   Other instance types will work, but with lesser performance. If you choose an instance type other than the recommended ones, make sure to run the :ref:`scylla_setup <system-configuration-scripts>` script.
+
+#. Launch a ScyllaDB instance on GCP with the ``gcloud``, using the following syntax:
+
+   .. code-block:: console
+      :substitutions:
+
+        gcloud compute instances create <name of new instance> --image |IMAGE-NAME| --image-project |PROJECT-NAME| --local-ssd interface=nvme --zone <GCP zone - optional> --machine-type=<machine type>
+
+
+   For example:
+
+   .. code-block:: console
+      :substitutions:
+
+        gcloud compute instances create scylla-node1 --image |IMAGE-NAME| --image-project |PROJECT-NAME|--local-ssd interface=nvme --machine-type=n1-highmem-8
```

Comment:
I've replaced the machine type type used on the website (n1-standard-8) with one of the currently recommended instance types. See https://docs.scylladb.com/master/getting-started/cloud-instance-recommendations.html#google-compute-engine-gce.